CHANGELOG.md

```diff
@@ -10,14 +10,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.11.0[[v0.11.0_log]
+--------------
+
 ### Added
-- OE SDK Release packages can now be built on non-SGX and non-FLC machines.
+- Open Enclave SDK Release packages can now be built on non-SGX and non-FLC machines.
 - Support for arbitrarily large thread-local data for SGX machines.
+- Experimental support for OpenSSL inside enclaves has been added while building the SDK from source.
+     - Use BUILD_OPENSSL flag while compiling the SDK
+     - [OpenSSLSupport.md](docs/OpenSSLSupport.md) documents supported options and configuration needed to OpenSSL inside an enclave.
 
 ### Changed
 - Rename the custom claims buffer added by oe_get_evidence from "custom_claims" to "custom_claims_buffer". Likewise, replace the `OE_CLAIM_CUSTOM_CLAIMS` definition for this name with `OE_CLAIM_CUSTOM_CLAIMS_BUFFER`.
+- Building SDK from source		
+      - HAS_QUOTE_PROVIDER cmake option has been removed. This is a continutation of the work in previous release to allo the same build of OE SDK to run on both FLC and non FLC machines."
+      - Intel SGX EnclaveCommonAPI pacakges are no longer needed to build the SDK.
 
 ### Removed
+- COMPILE_SYSTEM_EDL has been removed.
 
```

Comment:
Please add this to "Remove" list:

- The following SGX attestation format ID is not yet availble:
  - `OE_FORMAT_UUID_SGX_ECDSA_P384`